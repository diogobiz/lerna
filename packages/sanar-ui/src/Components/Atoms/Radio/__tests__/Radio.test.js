
import React from 'react'
import TestRenderer from 'react-test-renderer'

import ESRadio from '../'

it('renders correctly', () => {
    const component = (
        <ESRadio>
            <h1>ESRadio</h1>
        </ESRadio>
    )

    const tree = TestRenderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
})
