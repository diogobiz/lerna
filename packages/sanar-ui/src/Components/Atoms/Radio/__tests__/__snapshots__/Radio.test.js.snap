// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<label
  className="es-radio ant-radio-wrapper"
>
  <span
    className="ant-radio"
    style={Object {}}
  >
    <input
      checked={false}
      className="ant-radio-input"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      type="radio"
    />
    <span
      className="ant-radio-inner"
    />
  </span>
  <span>
    <h1>
      ESRadio
    </h1>
  </span>
</label>
`;
